bip-0053.mediawiki

```diff
@@ -51,7 +51,7 @@ is not a valid transaction according to network consensus rules.
 Other peers on the network receive the valid block containing T<sub>0</sub> and T<sub>1</sub>
 and add the block to their blockchain. Peers that receive the invalid block before the valid block
 will never come to consensus with their peers due to the malicious user finding a collision
-within the block's Merkle root. Finding this collision is approximately 22 bits worth of work.<ref>[https://github.com/Christewart/bips/blob/2024-12-20-64bytetxs/bip-XXXX/2-BitcoinMerkle.pdf to produce a block having a Merkle root that
+within the block's Merkle root. Finding this collision is approximately 22 bits worth of work.<ref>[https://github.com/Christewart/bips/blob/2024-12-20-64bytetxs/bip-0053/2-BitcoinMerkle.pdf to produce a block having a Merkle root that
```

Comment:
While those links work at the time, a relative link would make more sense to me, because it would work now, and still link to the correct document if it gets updated later.

```suggestion
within the block's Merkle root. Finding this collision is approximately 22 bits worth of work.<ref>[bip-0053/2-BitcoinMerkle.pdf to produce a block having a Merkle root that
```